tests/testthat/test.R

```diff
@@ -1,8 +1,8 @@
+test_that("gitcreds handles env var set to empty string", {
+  orig <- Sys.getenv("GITHUB_PAT_GITHUB_ACME_COM")
+  on.exit(Sys.setenv(GITHUB_PAT_GITHUB_ACME_COM = orig), add = TRUE)
```

Comment:
I would much prefer to use withr, due to the fiddliness of setting/resetting env vars, because this doesn't really handle all scenarios on all OSes.

Reply:
you can do it like this: https://github.com/r-lib/gitcreds/blob/0ebc53fde7d2dd5c3875c666ed352756cbaac988/R/git-auth.R#L653-L654